File: Scripts/GPIO/gpio.py
#Este script configura el BCM_GPIO 16 usando python
#EL BCM_GPIO 17 corresponde con la terminal 36 del conector
#de 40 pines de la raspberry

import RPi.GPIO as GPIO
import time

PIN = 36

#Se inicializa los GPIOS
GPIO.setmode( GPIO.BOARD )
#Se configura el BCM_GPIO 16 como salida usando la terminal 36
GPIO.setup( PIN, GPIO.OUT )

#Se escribe un valor digital al BCM_GPIO 16
while( True ):
    GPIO.output( PIN, GPIO.HIGH )
    time.sleep( 1 )
    GPIO.output( PIN, GPIO.LOW )
    time.sleep( 1 )

File: Scripts/GPIO/gpio.c
/* @brief:
 * Este programa configura el BCM_GPIO 16 usando la libreria wiringPi
 * El GPIO 16 corresonde con el pin 27 en wiring Pi
 * compilar: gcc gpio.c -o gpio -lwiringPi 
 * */

#include<stdio.h>
#include<unistd.h>
#include<wiringPi.h>

#define PIN 27

int main( void ){
	// Inizializa la libreria wiring Pi
	wiringPiSetup();
	// Se configura el GPIO 16 como salida
	pinMode( PIN, OUTPUT );

	// Se escribe un valor digital al GPIO
	while( 1 ){
		digitalWrite( PIN, HIGH );
		sleep( 1 );
		digitalWrite( PIN, LOW );
		sleep( 1 );
	}
       return 0;	
}
